src/Lkl.cc

```diff
@@ -585,6 +593,7 @@ Double_t Lkl::MinimizeLkl(Double_t g,Bool_t gIsFixed,Bool_t isVerbose,Bool_t for
   // call minimization (with requested verbosity)
   Int_t strategy = (!gIsFixed)*2; // when it's fixed, we can relax the strategy since there is no interest in errors
   Double_t lkl = CallMinimization(g,isVerbose,strategy);
+  if(lkl>8e99) return -1000.;
```

Comment:
why??

Reply:
It's linked to line 251: if (TMath::Abs(Lkl::MinimizeLkl() + 1000.) < 1.e-6) return 0;
This was a trick we introduced some years ago to allow gLike to exit properly in this case (otherwise it's crashing in the next step). We said we will change it for something better the day where we have a proper error handling system...
Btw line 560 also has "return -1000.;" (just in case you decide to tackle this now :-) ).

Reply:
Ok, changed, for the time being, the -1000 to the new gLklValOverflow value